### PR TITLE
Add sprites to specie specific boxes

### PR DESCRIPTION
**Pull request by datlo**
Y'all forgot these
n2
plasma

Apparently these are made by shazbot. Hmm. 🤔 

🆑
add: Added sprites to specie specific boxes made by shazbot
/🆑

### CODE REVIEW

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -75,7 +75,7 @@
 		return
 
 /obj/item/storage/box/survival_vox
-	icon_state = "box_civ"
+	icon_state = "box_vox"
```
Comment (KasparoVy) [inline code/game/objects/items/weapons/storage/boxes.dm]: Lmfao Vox box ftw

Comment (datlo) [reply code/game/objects/items/weapons/storage/boxes.dm]: had to resist the temptation of naming it vox_box and keep it consistent with the rest :(

**Comment by shazbot194** [comment]
I wonder what these boxes look like and where you got them

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 1 above)
Comment (taukausanake) [reply code/game/objects/items/weapons/storage/boxes.dm]: Should have done it anyways

**Comment by datlo** [comment]
spartan gave me the sprites, idk who originally made them. 
plasma
n2

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 1 above)
Comment (KasparoVy) [reply code/game/objects/items/weapons/storage/boxes.dm]: All good man, we just gonna read right-left on this one

**Comment by KasparoVy** [comment]
Licensed :ok:

**Comment by variableundefined** [comment]
Ok. I assum that means its license status is clear then.

**Pull request merged**